src/ui/public/courier/fetch/fetch.js

```diff
@@ -8,23 +8,32 @@ import ReqStatusProvider from './req_status';
 export default function fetchService(Private, Promise) {
 
   const requestQueue = Private(RequestQueueProvider);
-  const fetchThese = Private(FetchTheseProvider);
-
+  const immediatelyFetchThese = Private(FetchTheseProvider);
   const callResponseHandlers = Private(CallResponseHandlersProvider);
   const INCOMPLETE = Private(ReqStatusProvider).INCOMPLETE;
 
-  function fetchQueued(strategy) {
-    const requests = requestQueue.getStartable(strategy);
-    if (!requests.length) return Promise.resolve();
-    else return fetchThese(requests);
-  }
+  const debouncedFetchThese = _.debounce(() => {
+    const requests = requestQueue.get().filter(req => req.isFetchRequested());
```

Comment:
Oops, forgot to update this to the new `req.isFetchRequestedAndPending()` method that takes the requests' started state into account.